src/components/ReviewsList.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import ReviewCard from "./ReviewCard";
+
+const ReviewsList = () => {
+
+  const [reviewsList, setReviewsList] = useState([])
+
+  useEffect(() => {
+    fetch("https://fe-games-api-ed.herokuapp.com/api/reviews")
+    .then((res) => res.json()).then(({reviews}) => setReviewsList(reviews))
+  })
+  
+// 
+  return (
+    <div>
+      <h2>Reviews</h2>
+      <ul>
+        {reviewsList.map((review)=> {
+          return <ReviewCard review={review}/>
+        })}
+      </ul>
+    </div>
+  );
+};
+
+export default ReviewsList;
```

Comment:
what does it look like while we are waiting for the reviews to load? sometimes it could take longer.

let's add a loading indicator.  check out the data fetching notes